Harvest.Core/Services/AggieEnterpriseService.cs

```diff
@@ -179,13 +179,13 @@ private async Task GetPpmAccountManager(AccountValidationModel rtValue)
         /// </summary>
         /// <param name="financialSegmentString"></param>
         /// <returns></returns>
-        public async Task<FinancialOfficerDetails> GetFinancialOfficer(string financialSegmentString)
+        public Task<FinancialOfficerDetails> GetFinancialOfficer(string financialSegmentString)
         {
             //TODO: If AE can supply the query....
             var rtValue = new FinancialOfficerDetails();
             rtValue.FinancialOfficerId = null;
 
-            return rtValue;
+            return Task.FromResult(rtValue);

```

Comment:
I'm not seeing an obvious reason for this, and no code changed seems to be calling this.  Why can't we do this service the regular way?

Reply:
We probably will not implement this for harvest.
It can be deleted if you want.

Reply:
Just getting rid of a compiler complaint about marking a function as async without doing any async. I would have changed the return type from Task<FinancialOfficerDetails> to just FinancialOfficerDetails, but it turned out to be declared in an interface.